helpers/calendar/events.py

```diff
@@ -90,3 +92,25 @@ def get_all_room_schedules(self, query, start_date, end_date):
                 all_events.append(current_event)
 
         return all_events, all_dates
+
+    def check_event_status(self, info, **kwargs):
+        try:
+            room_id = RoomModel.query.filter_by(
+                calendar_id=kwargs['calendar_id']).first().id
+            if EventsModel.query.filter_by(
```

Comment:
Why do we need to use the .count() method yet we are just querying one event object?

Reply:
without the count, it'll return the query, which will always be true regardless of what it returns. so to get the object I added the count() and checked if it appear more than twice. The other alternative would be to use .one() but that would raise an error of `No row was found` when the query returns no rows. And the failing tests are not on my branch, I'll have to rerun the build till they pass.... @Bonifase what would suggest I use instead?

Reply:
@Bonifase the tests are fixed